lib/msbuild-finder.js

```diff
@@ -57,13 +57,25 @@ module.exports.find = function (options) {
     throw new PluginError(constants.PLUGIN_NAME, 'No MSBuild Version was supplied!');
   }
 
-  if (version === '12.0' || version === '14.0') {
+  if (version === '12.0' || version === '14.0' || version === '15.0') {
     // On 64-bit systems msbuild is always under the x86 directory. If this
     // doesn't exist we are on a 32-bit system. See also:
     // https://blogs.msdn.microsoft.com/visualstudio/2013/07/24/msbuild-is-now-part-of-visual-studio/
     var pathRoot = env_var_dir || path.join('C:', is64Bit ? 'Program Files (x86)' : 'Program Files');
     var x64_dir = is64Bit ? 'amd64' : '';
-    return path.join(pathRoot, 'MSBuild', version, 'Bin', x64_dir, 'MSBuild.exe');
+    var msbuildRoot = pathRoot
+
+    if(version === '15.0')
```

Comment:
Aren't these additional possible paths to the msbuild executable? If so it should fallback to the path in line 82 if both paths aren't available.

I've only found [this article](https://docs.microsoft.com/de-de/visualstudio/msbuild/what-s-new-in-msbuild-12-0), but none informations about where msbuild 15 is located, so I could be wrong.